src/Indicator.vala

```diff
@@ -43,14 +43,15 @@ namespace WingpanelMonitor {
         }
 
         construct {
+/*
             var provider = new Gtk.CssProvider ();
             provider.load_from_resource ("/com/github/plugarut/wingpanel-monitor/icons/Application.css");
             Gtk.StyleContext.add_provider_for_screen (
                 Gdk.Screen.get_default (),
                 provider,
                 Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION
             );
-
+*/
```

Comment:
@vjr Could you please remove the unused code ?

Reply:
> @vjr Could you please remove the unused code ?

@andirsun  done!